src/lib.rs

```diff
@@ -285,8 +285,8 @@ impl fmt::Display for Error {
                     io::ErrorKind::NotFound => {
                         let crate_name =
                             std::env::var("CARGO_PKG_NAME").unwrap_or_else(|_| "sys".to_owned());
-                        let instructions = if cfg!(target_os = "macos") || cfg!(target_os = "ios") {
-                            "Try `brew install pkg-config` if you have Homebrew.\n"
+                        let instructions = if cfg!(target_os = "macos") {
+                            "Try `brew install pkgconf` if you have Homebrew.\n"
                         } else if cfg!(unix) {
```

Comment:
Unfortunately this now triggers the `cfg!(unix)` condition, which is designed for BSD/Linux :slightly_smiling_face: 

```suggestion
                        let instructions = if cfg!(target_os = "macos") {
                            "Try `brew install pkgconf` if you have Homebrew.\n"
                        } else if cfg!(target_os = "ios") {
                            "" // iOS cross-compilation requires a custom setup, no easy fix
                        } else if cfg!(unix) {
```

Reply:
[Homebrew also supports Linux now](https://formulae.brew.sh/formula/pkgconf#default), so this entry could be added too.